src/video/groupnetcamview.h

```diff
@@ -31,26 +34,23 @@ class GroupNetCamView : public GenericNetCamView
 public:
     GroupNetCamView(int group, QWidget* parent = 0);
     void clearPeers();
-    void addPeer(int peer, const QString& name);
-    void removePeer(int peer);
-
-public slots:
-    void groupAudioPlayed(int group, int peer, unsigned short volume);
+    void addPeer(ToxPk peer, const QString& name);
+    void removePeer(ToxPk peer);
 
 private slots:
     void onUpdateActivePeer();
-    void friendAvatarChanged(int FriendId, const QPixmap& pixmap);
+    void friendAvatarChanged(int friendId, const QPixmap& pixmap);
 
 private:
     struct PeerVideo
     {
         LabeledVideo* video;
     };
 
-    void setActive(int peer = -1);
+    void setActive(ToxPk peer = ToxPk{});
```

Comment:
Since `setActive` do nothing for `ToxPk{}` why it even have a default value?

Reply:
I'm not sure this function is used at the moment, can probably be scrapped, but not in this PR